accounts/forms.py

```diff
@@ -214,8 +214,13 @@ def __init__(self, *args, **kwargs):
         super(FsAuthenticationForm, self).__init__(*args, **kwargs)
         self.error_messages.update({
             'inactive': mark_safe(_("You are trying to log in with an inactive account, please <a href=\"%s\">activate "
-                                    "your account</a> first." % reverse("accounts-resend-activation")))
+                                    "your account</a> first." % reverse("accounts-resend-activation"))),
+            'invalid_login': _(
+                "Please enter a correct username/email and password. Note that both "
+                "fields may be case-sensitive."
```

Comment:
Usernames are emails are case-insensitive so I guess password are the only case-sensitive. I just copied the default message here and added the /email part without questioning the other part. Maybe we should just put "Note that passwords are case sensitive" (I just checked, they are).